frontend/app/src/features/matching/api/useGetQuestionFilters.ts

```diff
@@ -0,0 +1,28 @@
+import { backendApi } from "@/lib/axios";
+import { makeSuccessResponseSchema } from "@/lib/api";
+import { useQuery } from "react-query";
+import { z } from "zod";
+import { API_ENDPOINT } from "@/constants/api";
+
+const GET_QUESTION_FILTERS = "question-filters";
+
+const getQuestionFiltersSchema = makeSuccessResponseSchema(
+  z.object({
+    Easy: z.array(z.string()),
+    Medium: z.array(z.string()),
+    Hard: z.array(z.string()),
+  }),
+);
+
+const getQuestionFilters = async () => {
+  const { data } = await backendApi.get(API_ENDPOINT.QUESTIONS_GET_FILTERS);
```

Comment:
the `QUESTIONS_GET_FILTERS` constant is missing a `/` at the start

Reply:
1c384783e3033a0b73dc3d9dc28efae685db3a86